### PR TITLE
Android: Add various media permissions

### PR DESCRIPTION
**Pull request by Summersay415**
Adds various media permissions added in Android 13 which are replaced READ_EXTERNAL_STORAGE

### CODE REVIEW

File: platform/android/doc_classes/EditorExportPlatformAndroid.xml [hunk 1]
```diff
@@ -154,6 +154,9 @@
 		<member name="permissions/access_mock_location" type="bool" setter="" getter="">
 			Allows an application to create mock location providers for testing.
 		</member>
+		<member name="permissions/access_media_location" type="bool" setter="" getter="">
```
Comment (m4gr3d) [inline platform/android/doc_classes/EditorExportPlatformAndroid.xml:157]: The entry should come before `access_mock_location`

File: platform/android/export/export_plugin.cpp [hunk 2]
```diff
@@ -69,6 +69,7 @@ static const char *android_perms[] = {
 	"ACCESS_FINE_LOCATION",
 	"ACCESS_LOCATION_EXTRA_COMMANDS",
 	"ACCESS_MOCK_LOCATION",
+	"ACCESS_MEDIA_LOCATION",
```
Comment (m4gr3d) [inline platform/android/export/export_plugin.cpp:72]: The entry should come before `access_mock_location`

**Review by m4gr3d — APPROVED** [review]

File: platform/android/export/export_plugin.cpp (hunk 2 above)
Comment (Summersay415) [reply platform/android/export/export_plugin.cpp:72]: forgot the alphabet 😅

**Review by m4gr3d — APPROVED** [review]
Thanks for the additions!

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**